### ISSUE TITLE
Logging out does not actually log you out

### ISSUE THREAD

**Issue by Dumbaz** [gitterHQ/gitter#859]
Because logging in uses a GitHub Account to auth, logging out does not work correctly, which may be an issue on public machines.

Specifically, one can just press log in again and will (still) have auth, same should be true for not only Gitter, but als GitHub.

To replicate just go to Gitter.im, Log in, Log out, press the login button again.

**Comment by MadLittleMods** [comment]
I think the problem stems from when signing into Gitter you also sign into GitHub. So when you logout of Gitter and press login, you are already signed in to GitHub can go straight through.

**Comment by Dumbaz** [comment]
Yes, that's exactly it.

Eric Eastwood <notifications@github.com> schrieb am Mi., 16. Sep. 2015
13:05:

**Comment by grahamperrin** [comment]
Is this still an issue?

**Comment by MadLittleMods** [comment]
@grahamperrin Yes. To rephrase a previous reply, https://github.com/gitterHQ/gitter/issues/859#issuecomment-140709295: When you press sign out on Gitter, you are signed out of Gitter, but you are still signed in to GitHub. Next time you press sign in, you will automatically get put through to Gitter because you are already signed in on Gitter.

This is as expected but perhaps we can also link to a GitHub sign out page in a banner once redirected to the homepage, https://github.com/logout
